system_setup/common/helpers.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
```

Comment:
We should call that file wsl_util.py I guess, as it’s specific to WSL.

Reply:
Fixed.